handlers/file-base64.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * file-base64
+ * return a file as base64
+ * our Request handler.
+ */
+const fs = require("fs/promises");
+const Jimp = require("jimp");
+
+const ABBootstrap = require("../AppBuilder/ABBootstrap");
+// {ABBootstrap}
+// responsible for initializing and returning an {ABFactory} that will work
+// with the current tenant for the incoming request.
+const serviceKey = "file_processor.file-base64";
+
+module.exports = {
+   /**
+    * Key: the cote message key we respond to.
+    */
+   key: serviceKey,
+   inputValidation: {
+      uuid: { string: { uuid: true }, required: true },
+      mobile: { boolean: true, optional: true },
+   },
+
+   /**
+    * fn
+    * our Request handler.
+    * @param {obj} req
+    *        the request object sent by the
+    *        api_sails/api/controllers/file_processor/file-get.
+    * @param {fn} cb
+    *        a node style callback(err, results) to send data when job is finished
+    */
+   fn: async function handler(req, cb) {
+      req.log("file_processor.file-base64:");
```

Comment:
Sorry, I know this is nit-picky-ish, but it's just the way my mind thinks.  

You've already set `serviceKey` to this value, might as well use it here.